### ISSUE TITLE
Server not responding in Version 0.6

### ISSUE THREAD

**Issue by sidharthramesh** [wardle/hermes#12]
I've imported and indexed all the files using the jar in [v0.6](https://github.com/wardle/hermes/releases/tag/v0.6.0).

I skipped the compact step due to a Java Heap Space error.

Running the serve command brings up everything in the terminal as usual
```bash
$ java -jar hermes-v0.6.jar -d db serve

2021-04-20 00:41:16,060 [main] INFO  com.eldrix.hermes.core - starting terminology server  {:http/server {:port 8080, :svc #integrant.core.Ref{:key :terminology/service}, :join? true}, :terminology/service {:path db}}
2021-04-20 00:41:16,218 [main] INFO  com.eldrix.hermes.terminology - hermes terminology service opened  "db" {:version 0.4, :store "store.db", :search "search.db", :created "2021-04-20T00:33:18.951033", :releases ("SNOMED Clinical Terms version: 20200731 [R] (July 2020 Release)")}
2021-04-20 00:41:16,219 [main] INFO  com.eldrix.hermes.server - starting server on port  8080
2021-04-20 00:41:16,447 [main] INFO  org.eclipse.jetty.server.Server - jetty-9.4.z-SNAPSHOT; built: 2019-04-29T20:42:08.989Z; git: e1bc35120a6617ee3df052294e433f3a25ce7097; jvm 11.0.8+10-post-Ubuntu-0ubuntu120.04
2021-04-20 00:41:16,488 [main] INFO  org.eclipse.jetty.server.Server - Started @4113ms
``` 

But there is no response when making a request.

```
Error: Couldn't connect to server
```

On chrome visiting http://localhost:8080
```
localhost refused to connect
```

**Comment by sidharthramesh** [comment]
Okay, just tested with Windows Java and there is no issue. It's only when using Java in WSL2.

**Comment by sidharthramesh** [comment]
I will test it with Docker and let you know soon

**Comment by sidharthramesh** [comment]
With Docker I get `ERR_EMPTY_RESPONSE`. I haven't compiled it from source, and this is just the .jar. 
I'm using it with the openJDK 8 image:
```yml
version: "3"
services:
  hermes:
    image: openjdk:8
    volumes:
      - ./db:/db/snomed.db
      - ./hermes-v0.6.jar:/usr/hermes.jar
    command: "java -jar /usr/hermes.jar -d /db/snomed.db -p 8083 serve"
    ports:
      - 8083:8083
    container_name: hermes
```

It works on Windows without any problems. But, on Linux there seems to be a problem. However, I've only tested it on the Linux subsystem within my Windows machine. The previous version I was using (0.1.0) didn't have this issue.

**Comment by wardle** [comment]
I've tested on mac os x and linux, and you've tested on Windows - so it is only not working on WSL? I've not tried on WSL I'll have to do some detective work - I do have a WSL on windows on one machine here so will give it a try. But, I have a feeling I might have removed the 0.0.0.0 from the configuration - I should set up a better configuration when I get chance and allowing binding to a specific host (and to allow CORS etc config). My suspicion is that is the cause of the issue.

**Comment by wardle** [comment]
Yes. My fault - 0f9cc66f4d3d9a42a8ee12b5bfc9548beb1a0852 removed the binding to 0.0.0.0   I'd prefer to make this a configuration option rather than a default.

**Comment by wardle** [comment]
Fixed in f93ebba99fbc7e99972fb75357b7673bad701f41 and release [v0.6.1](https://github.com/wardle/hermes/releases/tag/v0.6.1) pending fix via #14

**Comment by wardle** [comment]
Did this work?

And can you let me know whether you still had issues regarding heap if you used 
```
java -Xmx8g -jar hermes.jar --db snomed.db compact
```

to increase the amount of heap allocated to the JVM?

**Comment by wardle** [comment]
Sidharth : This is now fixed in 4c9d10b9393976cbd9efad2272114f975f942161 as you can specify a bind-address at the command line. It would be helpful to know if you still have the heap issue if you specify -Xmx8g as above to increase the amount of heap allocated to the JVM?

**Comment by sidharthramesh** [comment]
Hey Mark! Thank you for the update. And yes, the heap issue was solved after setting it to 8gb.
